codebrag-service/src/main/scala/com/softwaremill/codebrag/repository/Repository.scala

```diff
@@ -11,17 +15,25 @@ trait Repository extends Logging with RepositorySnapshotLoader with RepositoryDe
   def repoData: RepoData
   def repo: org.eclipse.jgit.lib.Repository
   val repoName = repoData.repoName
+  val config: CodebragConfig = new CodebragConfig {
+    def rootConfig = ConfigFactory.load()
+  }
```

Comment:
Good point, brings more simplicity to the code while reading. Going to push the modification.